labs-ios-starter/Profiles/View Controllers/ProfileListViewController.swift

```diff
@@ -28,7 +28,11 @@ class ProfileListViewController: UIViewController {
     // MARK: - Private Methods
     
     private func refresh() {
-        profileController.getAllProfiles {
+        profileController.getAllProfiles { error in
+            if let error = error {
+                self.presentAuthError(error: error)
+                return
+            }
             self.tableView.reloadData()
```

Comment:
This line might not be ready to run when it gets called without a semaphore or dispatch or something to handle the timing

Reply:
its in the getAllProfiles completion